xds/internal/env/env.go

```diff
@@ -0,0 +1,32 @@
+/*
+ *
+ * Copyright 2020 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// Package env acts a single source of definition for all environment variables
+// related to the xDS implementation in gRPC.
+package env
+
+const (
+	// BootstrapFileName is the name of the environment variable which holds the
+	// name of the bootstrap file.
+	BootstrapFileName = "GRPC_XDS_BOOTSTRAP"
```

Comment:
This change is OK as is.

I was thinking if we can move `os.Getenv` to this package as well. Like https://github.com/grpc/grpc-go/blob/master/internal/envconfig/envconfig.go

But the tests needs the set/unset them, they need the names.

Maybe export both? And only use the env names in the tests?

Reply:
> I was thinking if we can move os.Getenv to this package as well. Like https://github.com/grpc/grpc-go/blob/master/internal/envconfig/envconfig.go

Cool. I also initially wanted to define variables in this package which directly represent the value of the environment variable so that code can directly use them. I couldn't think of what is being done in the `envconfig` package. So, I discarded that idea. But now that I know how to do this in a succinct way, i can adopt the same approach.

> Maybe export both? And only use the env names in the tests?

Tests could use the names, but the actual code would still be using the var exported by the package. So, tests changing the environment variable through a call to `os.Setenv()` will not reflected in the exported var. Would it be OK for the tests to directly set the exported vars from this package?

Reply:
Ping ...

Reply:
Switched it to the same pattern as `envconfig`. PTAL. Thanks.